src/sheets/quadrone/item/parts/Sidebar.svelte

```diff
@@ -157,7 +179,7 @@
   </div>
 
   <!-- Item States -->
-
+  <!-- TODO: Make into snippets, stack into array, and don't render if there are no state pills. -->
```

Comment:
Get this done before approving.

Reply:
I changed my mind. The sidebar is getting a little too disjointed with the code in 3 places (script, loop, snippets at bottom) instead of 1.

Maybe it's something I need to get used to and then figure out a tidy ;) way to code it. Will wait for now.